ports/libdatachannel/vcpkg.json

```
@@ -1,9 +1,9 @@
 {
   "name": "libdatachannel",
-  "version-semver": "0.17.11",
+  "version-semver": "0.18.2",
   "description": "libdatachannel is a standalone implementation of WebRTC Data Channels, WebRTC Media Transport, and WebSockets in C++17 with C bindings for POSIX platforms (including GNU/Linux, Android, and Apple macOS) and Microsoft Windows.",
   "homepage": "https://github.com/paullouisageneau/libdatachannel",
-  "license": "LGPL-2.1-only",
+  "license": "MPL-2.0",
```

```suggestion
  "license": "MPL-2.0",
```
Why amend the license？

---

The license has been updated upstream, please see for details.https://github.com/paullouisageneau/libdatachannel/commit/a0f3f01bf5fdf9336bef830879e5bec07a8bca73